### DISCUSSION TITLE
Reverse proxy in a subdir

### DISCUSSION THREAD

**Discussion by yuritsuki** [ONLYOFFICE/Docker-DocumentServer#702] [Q&A]
Is it possible to make DocServer work in a subdirectory? My application is running on example.com, and I need OnlyOffice to function in a subdirectory, specifically example.com/docserver, for API/WOPI. When I attempted to proxy /docserver/ to OnlyOffice via Nginx, I encountered an issue: When the iframe with OnlyOffice tries to fetch its resources, it fetches them from example.com/[version]~oou/... instead of example.com/docserver/[version]~oou/. I even attempted to proxy /[version]~oou/, oo loaded but it led to a new set of problems, such as: "The file content corresponds to one of the following formats: pdf/djvu/xps/oxps, but the file has an inconsistent extension: docx." This issue arises when I try to edit a docx file via the API.

**Comment by dreua** [comment]
Die you find a solution or end up using a subdomain instead?

**Comment by yuritsuki** [comment]
No, I duplicated it to oo forums and got the first answer, so it's better to continue there https://forum.onlyoffice.com/t/reverse-proxy-in-a-subdir/7578

**Reply by dreua** [reply]
It's always nice to leave a link for the next person with the same issue 👍
